packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.1.1
```

Comment:
This should not be a version bump as it does not affect customers of the package. It should use `NEXT`, per my earlier comment.